src/pages/achievement/control/subcomponents/editorTools/AchievementTemplate.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+  AchievementAbility,
+  AchievementItem,
+  AchievementModalItem
+} from '../../../../../commons/achievement/AchievementTypes';
+
+export const modalTemplate: AchievementModalItem = {
+  modalImageUrl:
+    'https://source-academy-assets.s3-ap-southeast-1.amazonaws.com/images/sourceAcademy%40x2.png',
+  description: 'Modal Description here',
+  completionText: 'Completion Text here'
+};
+
+export const achievementTemplate: AchievementItem = {
+  id: 0,
+  title: '',
+  ability: AchievementAbility.CORE,
+  deadline: new Date(),
+  isTask: false,
+  release: new Date(),
+  prerequisiteIds: [],
+  goals: [
+    {
+      goalId: 0,
+      goalText: 'Sample GOal',
+      goalProgress: 0,
+      goalTarget: 0
+    }
+  ],
+  position: 0,
+  backgroundImageUrl:
+    'https://www.publicdomainpictures.net/pictures/30000/velka/plain-white-background.jpg',
```

Comment:
I'm not sure about having to use a remote image just for plain white background. Is there simpler way of doing this?

Reply:
Done ca57ca4